json-files/test-map.json

```diff
@@ -0,0 +1,162 @@
+[
```

Comment:
I know this isn't part of the code review, but I love the creativity of the story here 😁